clientapp/src/components/Card.jsx

```diff
@@ -262,4 +273,53 @@ export class Card extends React.Component{
     getShareForm() {
         return <ShareForm id={this.props.messageId} timeStamp={this.props.messageTimestamp} />
     }
+
+    shouldShowEditMenu()
+    {
+        return isTracerView() // && this.props.signedIn
```

Comment:
we should make sure user is signed in before letting them edit

Reply:
I have passed in the signedIn field in LandingPage to Card. You can check my line 133 in LandingPage.jsx in this PR. I haven't added it here, since it is easier to debug. I will remove comment once we can resolve the menu can't been seen issue.